### PR TITLE
Add unit tests to Chapter 9

### PR DESCRIPTION
**Pull request by pavpanchekha**
These tests found a few big bugs:

 - `nodes_to_handles` was only sometimes using `id(elt)` for the key, leading to many DOM APIs failing
 - `__runHandlers` had its arguments in the wrong order, leading to no listeners ever running

### CODE REVIEW

File: src/lab9-tests.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
+Tests for WBE Chapter 9
+=======================
+
+Chapter 9 (Running Interactive Sctipts) introduces JavaScript and the DOM API,
```
Comment (chrishtr) [inline src/lab9-tests.md:4]:
```suggestion
Chapter 9 (Running Interactive Scripts) introduces JavaScript and the DOM API,
```

File: src/lab9-tests.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+test things:
+
+    >>> div = b.tabs[0].nodes.children[0].children[0]
+    >>> js.run("var div = document.querySelectorAll('div')[0]")
+    >>> js.run("div.addEventListener('test', function(e) { console.log('Listener ran!')})")
+    >>> js.dispatch_event("test", div)
+    Listener ran!
+    False
+
+The `False` is from our `preventDefault` handling.
```
Comment (chrishtr) [inline src/lab9-tests.md:215]:
```suggestion
The `False` is from our `preventDefault` handling (we didn't call it).
```

**Review by chrishtr — APPROVED** [review]

**Comment by chrishtr** [comment]
Looks great!

**Comment by pavpanchekha** [comment]
Good to merge?

**Pull request merged**